.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,13 @@
-<!-- Thank you for creating a pull request!
+<!-- Thank you for creating a pull request! 
 
-Please go through [our contribution guide][CONTRIBUTING.md] to hopefully get your changes merged quicker.
+##### Checklist before submitting
```

Comment:
Should we still keep this line?

Reply:
In my first opinion, if we will add another check, it is better to keep it. But, currently there are no candidates, so I reverted it.